src/scss/lib/_string.scss

```diff
@@ -0,0 +1,42 @@
+// Converts a camelCase $string into a kebab-case one.
+//
+// @param {String} $string - String in camelCase
+//
+// @return {String} - String in kebab-case
+// @throws {Error}
+//
+// @access public
+
+@function camelToKebabCase($string) {
```

Comment:
`MaSeLaPrimaLetteraFosseMaiuscola` -> `-ma-se-la-prima-lettera-fosse-maiuscola` è un caso che dobbiamo gestire con o senza il `-` iniziale?

Reply:
@eFFeeMMe appena pushato una modifica con un controllo maggiore per sicurezza, quindi adesso in questi casi:
```
camelToKebabCase("CoseBelle");
camelToKebabCase("coseBelle");
camelToKebabCase("cose Belle");
camelToKebabCase("Cose Belle");
```
il risultato sarà sempre:
`cose-belle`

detto questo per me il `camelCase` e il `PascalCase` sono abbastanza definiti e differenti, ma sentiamo pure un parere dagli altri :D